docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -116,6 +116,26 @@ Custom yaml settings are silently ignored by {fleet}.
 Settings with input blocks, such as Max agents are still effective.
 ====
 
+[[known-issue-issue-2249-8.6.2]]
+.{agent} does not load {fleet} configuration when upgrading to {agent} version 8.6.x from version 8.2 or earlier
```

Comment:
```suggestion
.{agent} does not load {fleet} configuration when upgrading to {agent} version 8.6.x from version 8.2.x or earlier
```

For consistency we should use 8.2.x to ensure people assume this applies to all 8.2.x version. I'll just make this comment one time but it applies in multiple places.